test/parallel/test-fs-access.js

```diff
@@ -57,6 +57,10 @@ assert.throws(function() {
   fs.access(__filename, fs.F_OK);
 }, /callback must be a function/);
 
+assert.throws(function() {
+  fs.access('foo\u0000bar', fs.F_OK, {});
+}, /callback must be a function/);
```

Comment:
test/parallel/test-fs-null-bytes.js is arguably a better place for this but I'll leave that up to you.